api/forgot_test.go

```diff
@@ -38,6 +38,26 @@ func TestForgotResponds(t *testing.T) {
 				"password": "myN3wpa55w0rd",
 			},
 		},
+		{
```

Comment:
Need to add tests to verify the new behaviour of method `passwordReset` when requester is a patient